packages/components/organisms/f-loyalty/src/components/Terms.vue

```diff
@@ -0,0 +1,36 @@
+<template functional>
+    <div :class="$style['c-loyalty-terms']">
+        <a
+            :class="$style['c-loyalty-terms-link']"
+            data-test-id="terms-and-conditions"
+            :href="parent.$t('termsUrl')">
+            {{ parent.$t("termsText") }}
+        </a>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'StampCardsTerms'
```

Comment:
No need to specify the name property in the components — it'll be added automatically by the build tooling.